### ISSUE TITLE
FR: Add `<app_name>latest` option when using `-p/--plists`

### ISSUE THREAD

**Issue by apizz** [carlashley/appleloops#9]
Given the recent certificate changes on Apple pkgs, I was going through to redownload all the loops.  I do this from my workstation which does not have any of the apps installed.  As a result, I need to use the `-p / --plists` flag and choose from the available list which versions I want.

What I ran initially and the output (side note, it appears expected in the error message is misspelled):
```
appleloops -p /path/to/garageband1021.plist /path/to/logicpro1050.plist /path/to/mainstage340.plist -d /Volumes/appleloops -m -o
/usr/local/bin/appleloops: error: argument -p/--plists: excpected one argument: (choose from 'garageband1011', 'garageband1012', 'garageband1015', 'garageband1016', 'garageband1020', 'garageband1021', 'logicpro1021', 'logicpro1022', 'logicpro1023', 'logicpro1030', 'logicpro1032', 'logicpro1040', 'logicpro1042', 'logicpro1050', 'mainstage323', 'mainstage324', 'mainstage330', 'mainstage340')
```

The error reminded me that I just need to supply the app name and version, rather than the direct plist path, which got me thinking a small addition could improve this.

Overwhelmingly, I just want the latest available of each / all apps.  As such, my thought was that it would be helpful to add some shorthand to select the latest / highest version plist (currently `garageband1021`, `logicpro1050`, & `mainstage340`).  Something like `garagebandlatest`, `logicprolatest`, and `mainstagelatest`.

For me personally, `alllatest` would be helpful as well because I just want everything anyway.  So the end result when running appleloops could be:

`appleloops -p alllatest -d /Volumes/appleloops -m -o`

**Comment by carlashley** [comment]
This has been implemented in the [v3.0.9](https://github.com/carlashley/appleloops/releases/tag/v3.0.9) release.

**Comment by apizz** [comment]
Thank you! You're doing great work @carlashley
